libs/@guardian/libs/src/performance/serialise.ts

```diff
@@ -5,4 +5,4 @@ import type { TeamName } from '../logger/@types/logger';
 const SEPARATOR = ':';
 
 export const serialise = (team: TeamName, name: string, action?: string) =>
-	[team, name, action].filter(isNonNullable).join(SEPARATOR);
+	['gu', team, name, action].filter(isNonNullable).join(SEPARATOR);
```

Comment:
@mxdvl a minor suggestion which I can add if you agree:

I find a dot separator easier to visually scan as it adds lots of whitespace between the tokens.

e.g.

`dotcom:SetAdTargeting:import`

vs

`dotcom.SetAdTargeting.import`

I believe we will be transforming the separator to a BigQuery friendly `_` at transmission time regardless.

Reply:
I'm not too bothered by the separator but I'm not a fan of `.` just to make it clear that these have no relation to JS objects or BigQuery schemas/datasets at first glance.

Reply:
I’d actually pick the separator that the **least** readable, if we want to encourage consistency, and opt for a a deserialiser instead…

```ts
type Deserialiser = (measure: PerformanceMeasure) => { team: TeamName, name: string, action?: string, duration: number, start: number }
// or even just for a name…
type Deserialiser = (name: string) => { team: TeamName, name: string, action?: string }
```

Reply:
> I'm not too bothered by the separator but I'm not a fan of `.` just to make it clear that these have no relation to JS objects or BigQuery schemas/datasets at first glance.

I hadn't considered that someone would confuse them with JS objects but I guess its possible.

Reply:
> I’d actually pick the separator that the **least** readable, if we want to encourage consistency, and opt for a a deserialiser instead…
> 
> ```ts
> type Deserialiser = (measure: PerformanceMeasure) => { team: TeamName, name: string, action?: string, duration: number, start: number }
> // or even just for a name…
> type Deserialiser = (name: string) => { team: TeamName, name: string, action?: string }
> ```

I tend to look at performance metrics in dev tools quite a bit so this is less readable for me but happy to go with the consensus.